docs/external-node.md

```diff
@@ -561,3 +562,26 @@ interfaces will be added in the future.
 
 `ExternalNode` name must be unique in the `cluster` scope even though it is
 itself a Namespaced resource.
+
+## Known issues
+
+OVS configurations must be manually removed after the VM is rebooted, and then
+a manual restart on antrea-agent is needed. This is a workaround to resolve
+issue [#4122](https://github.com/antrea-io/antrea/issues/4122), which exists
+on both Linux and Windows if OVS is running as a service or a process.
+
+- To remove OVS configurations and restart antrea-agent on Linux,
+  ```shell
```

Comment:
Add line before  and after the shell section.

line1

```shell
example
```

line2

Reply:
done.